### PR TITLE
Resolved #44: Creating test to render JS files

### LINKED ISSUE #44: Testing Render JS files

### PR DESCRIPTION
**Pull request by MariuszZalew**

### CODE REVIEW

**Comment by MariuszZalew** [comment]
Waiting for review

File: test/render/source/select.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import select from "../../../src/render/source/select"
+
+describe('render/source/select', () => {
+    it('should match to Snapshot', () => {
+        const results = select([{
+            inAttr: "data-test",
+            id: "rock",
+            outAttr: 'kamienie',
+            label: "seed"
+        }], "lorem_{attr_in}_{id}_{attr_out}_{label}")
```
Comment (benedyktdryl) [inline test/render/source/select.test.js]: What is that string?

File: test/render/source/option.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import option from "../../../src/render/source/option"
+
+describe('render/source/option', () => {
+    it('should match to Snapshot', () => {
+        const results = option([
+            {index: "qwer", source: "1234"},
+            {index: "rock", source: "0987"}
+        ], "lorem_{key}_{value}")
```
Comment (benedyktdryl) [inline test/render/source/option.test.js]: Please make line space between `const` and other expressions in all files.

File: test/render/source/select.test.js (hunk 1 above)
Comment (MariuszZalew) [inline test/render/source/select.test.js]: "lorem_{attr_in}_{id}_{attr_out}_{label}" - it refers to second parameter of the tested function. It get's values from the firs parameter. I already run the test and it worked.

File: test/render/source/option.test.js (hunk 2 above)
Comment (MariuszZalew) [inline test/render/source/option.test.js]: already done it :)

File: test/render/source/select.test.js (hunk 1 above)
Comment (MariuszZalew) [inline test/render/source/select.test.js]: already fixed, please check it

**Pull request merged**